tests/adapter/mongo/test_case_events.py

```diff
@@ -1,36 +1,5 @@
-def test_diagnose(adapter, institute_obj, case_obj, user_obj, omim_term):
```

Comment:
This is redundant now, since there is a test that is calling the endpoint to modify diagnoses now, and it's testing more code that this test here